src/python/analyzer_executor/src/analyzer_executor_lib/analyzer_executor.py

```diff
@@ -389,9 +393,13 @@ def chunker(seq: List[BaseView], size: int) -> List[List[BaseView]]:
     return [seq[pos : pos + size] for pos in range(0, len(seq), size)]
 
 
-def emit_event(s3: S3ServiceResource, event: ExecutionHit) -> None:
+def emit_event(s3: S3ServiceResource, event: ExecutionHit, metadata: Meta) -> None:
     LOGGER.info(f"emitting event for: {event.analyzer_name, event.nodes}")
 
+    meta_dict = {
+        "trace_id": str(proto_uuid_to_pyuuid(metadata.trace_id)),
+    }
+
```

Comment:
The Python services don't communicate via protobuf right now - I'd rather handle that in another PR, and ideally post-nomad so that we can move a lot of things to RPC.